GPflowOpt/optim.py

```diff
@@ -175,21 +161,39 @@ def _optimize(self, objective):
                               nfev=points.shape[0],
                               message="OK")
 
+    def set_initial(self, initial):
```

Comment:
Yep, its the MCOptimizer/CandidateOptimizer which ignore this and issue a warning if you do so.